config/locales/zxcvbn/en.yml

```diff
@@ -0,0 +1,38 @@
+en:
+  zxcvbn:
+    feedback:
+      "Use a few words, avoid common phrases": Use a few words, avoid common phrases
+      "No need for symbols, digits, or uppercase letters": >-
+        No need for symbols, digits, or uppercase letters
+      "Add another word or two_ Uncommon words are better_": >-
```

Comment:
Yeah agree, the rest of our locale files follow the 100 char line limit though so I figured it'd be best to stick with that.